examples/deployment/README.md

```diff
@@ -0,0 +1,731 @@
+# Coherence Operator Deployment Example
+
+This example showcases how to deploy Coherence applications using the Coherence Operator.
+
+The following scenarios are covered:
+
+1. Installing the Coherence Operator 
+1. Installing a Coherence cluster 
+1. Deploying a Proxy tier
+1. Deploying an storage-disabled application 
+1. Enabling Active Persistence
+
+After the initial install of the Coherence cluster, the following examples 
+build on the previous ones by issuing a `helm apply` to modify
+the install adding additional roles.
+
+You can use `helm create` for any of the examples to install that one directly.
```

Comment:
Again, we use `kubectl` to create, update and delete Coherence clusters. Helm is only used to install the operator.